logs_http.module

```diff
@@ -93,146 +68,27 @@ function _logs_http_decode_exception($exception) {
  * devel_boot() registers this function as a shutdown function.
  */
 function logs_http_shutdown() {
-  if (!$events = logs_http_get_registered_events()) {
+  if (!LogsHttpLogger::isEnabled()) {
```

Comment:
It is a service, unless my definition is wrong over here:
https://github.com/Gizra/logs_http/pull/10/files#diff-61d4a8c453273c3ccb874a4c87290024R6